Nodejs/Product/Nodejs/Snippets/1033/Test/mochatest.snippet

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>mochatest</Title>
+      <Description>Code Snippet for a Unit Test using Mochajs</Description>
+      <Shortcut>mochatest</Shortcut>

```

Comment:
Let's split this one out into two snippets - ```describe``` and ```it```, and both should be a SurroundWith + Expansion. That way it's easier for people to drop it into their code. 

Don't bother with the require - this is the sort of snippet people might use multiple times in a file, so we don't want to repeat that code.